skimage/util/_coord_conv.py

```diff
@@ -0,0 +1,234 @@
+import numpy as np
+
+
+def xy_to_rc(points=None, image=None):
+    """Convert points' coordinates and/or image from `x, y` to `r, c` convention.
+
+        o--->....       o--->....
+        |   x   .       |   c   .
+        |       .  ==>  |       .
+        v y     .       v r     .
+        .........       .........
+
+    Parameters
+    ----------
+    points : (K, 2) ndarray, optional
```

Comment:
Hmm... I'm not sure if I understand how to convert nD coordinates (e.g. nD `x, y` coords into nD `r, c`). What should be the order of the basis vectors' in these spaces for >2 dims (ok, clear for `r, c` - we just enumerate them sequentially (r, c, ax2, ax3, ...), but what is it for `x, y`)?
The point above is the reason why I limited this feature to 2D only.